### DISCUSSION TITLE
I have 2 movie libraries

### DISCUSSION THREAD

**Discussion by mash2k3** [croneter/PlexKodiConnect#1904] [General]
I have 2 movie libraries in Plex from different locations
Under customize path there is only 1 movie section to replace path name
What do i need to do to replace path for second library?

**Comment by hamakavula** [comment]
You can 'separate' the libraries with a "proper Kodi-skin" I recommend 'Estuary MOD v2' by Kodinerds (https://repo.kodinerds.net/).

This is a very awesome adaptation of Kodi's original skin but with many, many improvements ,features and display options.
You can (easily) edit the main menu and all widgets, it takes a little 'getting used to' when changing the settings but gets the job done perfectly.

- Install **Kodi**
- Install **PKC**
- Install **Estuary MOD v2** _(for Matrix or Nexus) and go to;_
- Settings -> Skin settings -> Main Menu -> Edit Main Menu -> Start editing

It looks like this; (All red-underlined items are in a separate Movies- or Series-library configured in Plex).
![image](https://user-images.githubusercontent.com/17685480/218882609-7f9d1870-660f-46d6-b33f-fe4ee251f2d1.png)

You can also edit ALL widgets to your needs and create any type of widget-array this way.
![image](https://user-images.githubusercontent.com/17685480/218882891-b5732d51-4a54-4a87-81a1-7877e19f7fab.png)

Your Kodi-Home screen can be adapted to fit different libraries in separate menu's (or one combined, the Kodi default movie/serie-library display, like you have now).
![image](https://user-images.githubusercontent.com/17685480/218883314-3f253fa0-235b-483e-ad37-b82f43937ef3.png)

Also with this skin, Movie- and series artwork will be used in the best way possible. I tried many, many, many different skins but this one is by far the best and most stable. And (almost) everything can be configured in the settings.

Try it ...  and you will love it. Enjoy!

**Comment by croneter** [comment]
Also see https://github.com/croneter/PlexKodiConnect/wiki/Video-Nodes-and-Skins

**Comment by mash2k3** [comment]
My issue is not displaying my libraries in kodi. My issue is that i have 2 libraries in plex with different path ips but in plexkodiconnect settings you are only allowed 1 movie path customization.

**Reply by hamakavula** [reply]
I don't understand, please be more specific...

**Reply by mash2k3** [reply]
Here is a screenshot showing the settings of PKC, as you can see only 1 spot for movies so i can only customize a path for 1 library from plex. second library has a different ip, ends with 124.

![image](https://user-images.githubusercontent.com/1356023/219448389-792e6fce-30ea-464c-9696-d67bfbe2b6e9.png)

**Comment by jbelew** [comment]
I also have this issue. Movies are distributed across 4 different mount points --

```
192.168.0.17:/volume1/media    /mnt/media     
192.168.0.12:/mnt/HD_a2           /mnt/media-2    
192.168.0.13:/mnt/HD_a2           /mnt/media-3    
192.168.0.14:/mnt/HD_a2           /mnt/media-4
```

Seems that I'm limited to being able to do path substitution for only one mount point per media type.

**Comment by hamakavula** [comment]
Now it's clear... and indeed; only one path substitution is allowed in the PKC addon. 

So the path redirection/substition needs to be at the server/Plex side.

Plex, or the OS its running on, should be able reroute these paths internally I guess... then you can make PKC work with only one path substitution.

Wich OS are you using and how are these network shares indexed by Plex server now?
